src/util/wallet-utils/src/getAddress.ts

```diff
@@ -100,6 +103,27 @@ export const getAddressesXverse: getAddresses = async (params) => {
   return getWalletAddresses(result);
 };
 
+export const getAddressesFordefi: getAddresses = async (params) => {
+  const response = await request(
+    "wallet_connect",
+    {
+      addresses: [
+        AddressPurpose.Ordinals,
+        AddressPurpose.Payment,
+        AddressPurpose.Stacks,
+      ],
+    },
+    FORDEFI_PROVIDER_ID,
```

Comment:
Do we need to specify xverse provider id as well in `getAddressesXverse` above?

Reply:
No it's the default